__tests__/StarkwareLib.test.ts

```diff
@@ -5,9 +5,6 @@ import { DummyProvider } from './helpers/DummyProvider';
 import { Networks } from '../src/types';
 import Web3 from 'web3';
 
-// async ropsten calls can take >10k MS
```

Comment:
Since the default in node service base dev is 30s and this is higher, maybe we need to keep it?

Reply:
reverted

Reply:
i thought there would be a large buffer